src/os_user.eliomi

```diff
@@ -119,9 +119,6 @@ val is_complete : Os_types.User.t -> bool
 
 [%%server.start]
 
-val emails_of_user : Os_types.User.t -> string Lwt.t
-(** [emails_of_user user] returns the emails of the user [user]. *)
-
```

Comment:
Unwanted change?

Reply:
No, the signature appeared twice in the same file.